client/allocator/src/freeing_bump.rs

```diff
@@ -404,7 +411,21 @@ impl FreeingBumpHeapAllocator {
 		Header::Occupied(order).write_into(mem, header_ptr)?;
 
 		self.total_size += order.size() + HEADER_SIZE;
-		trace!("Heap size is {} bytes after allocation", self.total_size);
+
+		log::trace!(
+			target: LOG_TARGET,
+			"after allocation, total_size = {}, bumper = {}.",
+			self.total_size,
+			self.bumper,
+		);
```

Comment:
Was this added after your sample output in the top comment was made?

I would expect to see this in your sample output but don't currently see it.

Reply:
this is trace, I am only doing `debug` in my sample.